src/einsteinpy/tests/test_metric/test_schwarzschild.py

```diff
@@ -59,12 +59,17 @@ def test_calculate_trajectory(
     )
     _c, _scr = constant.c.value, schwarzschild_radius(M).value
     ans = ans[1]
-    testarray = (
-        (1 - (_scr / ans[:, 1])) * np.square(ans[:, 4])
-        - (np.square(ans[:, 5])) / ((1 - (_scr / ans[:, 1])) * (_c ** 2))
-        - np.square(ans[:, 1] / _c)
-        * (np.square(ans[:, 6]) + np.square(np.sin(ans[:, 2])) * np.square(ans[:, 7]))
-    )
+    testarray = list()
+    for ansi in ans:
```

Comment:
Can we make it for i in ans: ?

Reply:
@shreyasbapat Thanks for pointing that out. Could you make the changes in `test_calculate_trajectory()` in `test_kerr.py` ?

Reply:
you mean the same change. --"for i in ans"?

Reply:
Yups

Reply:
done this.